whipper/common/program.py

```diff
@@ -614,3 +617,12 @@ def writeLog(self, discName, logger):
         self.logPath = logPath
 
         return logPath
+
+    def writeToc(self, discName):
+        tocPath = '%s.toc' % discName
+        logger.debug('write .toc file to %s', tocPath)
```

Comment:
Let's use the present active: "writing .toc file to %s"

Reply:
As there's a similar `logger.debug` in `whipper/command/cd.py` maybe I should drop one of them? (redundant)